extensions/proposals/OES_EGL_image_external/extension.xml

```diff
@@ -44,6 +45,13 @@
 
   <idl xml:space="preserve">
 [NoInterfaceObject]
+interface WebGLVideoFrameInfo {
+  readonly attribute double currentTime;
+  readonly attribute unsigned long textureWidth;
+  readonly attribute unsigned long textureHeight;
+};
```

Comment:
Instead of adding this structure, consider adding these attributes to the video element itself.  currentTime, in particular, seems like it would have applications outside of WebGL.